.github/workflows/pull-request.yml

```
@@ -403,6 +404,26 @@
           docker stop talawa-admin-app-container
           docker rm talawa-admin-app-container
 
+  Test-Docusaurus-Deployment:
+    name: Test Deployment to https://admin-docs.talawa.io
+    runs-on: ubuntu-latest
+    needs: [Docker-Start-Check, Start-App-Without-Docker]
+    # Run only if the develop-postgres branch and not dependabot    
+    if: ${{ github.actor != 'dependabot[bot]' && github.event.pull_request.base.ref == 'develop-postgres' }}
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)